src/wrapper/state.rs

```diff
@@ -1091,6 +1146,17 @@ impl State {
     }
   }
 
+  pub fn to_str_in_place(&mut self, index: Index) -> Option<&str> {
+      let mut len = 0;
```

Comment:
I recommend a doc comment noting "Maps to `lua_tolstring`" and removing the current "omitted: lua_tolstring" comment. Also, minor, but there's an indentation error here.